### DISCUSSION TITLE
这个项目很久没有更新了，下一步有什么计划吗

### DISCUSSION THREAD

**Discussion by tbscer** [weihuajiang/WPF-Blocky#8] [General]
这个项目很久没有更新了，下一步有什么计划吗
我看到在gittee上也有托管，感兴趣的人还是挺多的

**Comment by weihuajiang** [comment]
这个是开源的通病吧，开发者很难有时间和精力去继续深入维护，我目前也同样也没法计划深入维护这个项目，也没有任何更新计划。如果你们有兴趣，可以pull request，进行更新和增加额外功能。
